pkg/webhook/resources/virtualmachine/mutator.go

```diff
@@ -465,3 +479,7 @@ func hostDevicesOvercommitNeeded(oldVM, newVM *kubevirtv1.VirtualMachine) bool {
 	}
 	return false
 }
+
+func isDedicatedCPU(vm *kubevirtv1.VirtualMachine) bool {
+	return vm.Spec.Template.Spec.Domain.CPU != nil && vm.Spec.Template.Spec.Domain.CPU.DedicatedCPUPlacement
```

Comment:
It seems the should be 

>...&& vm.Spec.Template.Spec.Domain.CPU.DedicatedCPUPlacement != nil && *vm.Spec.Template.Spec.Domain.CPU.DedicatedCPUPlacement

```
	// +optional
	DedicatedCPUPlacement *bool `json:"dedicatedCPUPlacement,omitempty"`
```

Reply:
Good catch, many thanks

Reply:
Correct me if I'm wrong. Looks like the DedicatedCPUPlacement field is `bool`, not `*bool`. 
https://github.com/harvester/harvester/blob/73c3584d264228463648e7fe7ba5d40fbf22cda8/vendor/kubevirt.io/api/core/v1/schema.go#L328

Reply:
You are right. I was confused by the kubevirt code, there was another `DedicatedCPUPlacement` defined as `*bool`...

```
vendor/kubevirt.io/api/instancetype/v1beta1/types.go:	DedicatedCPUPlacement *bool `json:"dedicatedCPUPlacement,omitempty"`

vendor/kubevirt.io/kubevirt/pkg/instancetype/instancetype.go:		vmiSpec.Domain.CPU.DedicatedCPUPlacement = *instancetypeSpec.CPU.DedicatedCPUPlacement
```